### PR TITLE
Add deprecation guide for ds.deprecated-initializers #2780

### PR DESCRIPTION
**Pull request by jenweber**
Closes https://github.com/emberjs/website/issues/2780

@locks

### CODE REVIEW

File: source/deprecations/ember-data/v2.x.html.md [hunk 1]
```diff
@@ -676,3 +676,36 @@ Using `JSONAPISerializer.payloadKeyFromModelName` to serialize the type of a
...
+
+export default {
+  name: 'websocketInit',
+  after: 'store',
+  initialize: initialize
+};
+```
+
+After:
+```javascript
```
Comment (locks) [inline source/deprecations/ember-data/v2.x.html.md]: Empty line before code block please!

File: source/deprecations/ember-data/v2.x.html.md [hunk 2]
```diff
@@ -676,3 +676,36 @@ Using `JSONAPISerializer.payloadKeyFromModelName` to serialize the type of a
...
+
+#### Unused Initializers
+
+###### until: 3.0.0
+###### id: ds.deprecated-initializers
+
+The Ember Data initializers `data-adapter`, `injectStore`, `transforms`, and `store` are no longer used, so they are being removed. Apps that depend on these for the ordering of their own custom initializers can substitute `ember-data` instead, without any change in functionality.
+
+Before:
+```javascript
```
Comment (locks) [inline source/deprecations/ember-data/v2.x.html.md]: Empty line before code block please!

File: source/deprecations/ember-data/v2.x.html.md [hunk 3]
```diff
@@ -676,3 +676,36 @@ Using `JSONAPISerializer.payloadKeyFromModelName` to serialize the type of a
 See [JSONAPISerializer.payloadKeyFromModelName for
 Resource](#toc_jsonapiserializer-payloadkeyfrommodelname-for-resource) for more
 information.
+
+#### Unused Initializers
+
+###### until: 3.0.0
+###### id: ds.deprecated-initializers
+
+The Ember Data initializers `data-adapter`, `injectStore`, `transforms`, and `store` are no longer used, so they are being removed. Apps that depend on these for the ordering of their own custom initializers can substitute `ember-data` instead, without any change in functionality.
```
Comment (locks) [inline source/deprecations/ember-data/v2.x.html.md]: Can you replace "Apps" with "Applications" and add a newline after the "removed." full stop?

Like so:

```
… removed.
Applications that …
```

Comment (jenweber) [inline source/deprecations/ember-data/v2.x.html.md]: Should there always be newlines at the end of sentences?

**Pull request merged**